example/src/main/java/org/xmtp/android/example/MainActivity.kt

```diff
@@ -182,6 +183,7 @@ class MainActivity : AppCompatActivity(),
 
     private fun showError(message: String) {
         val error = message.ifBlank { resources.getString(R.string.error) }
+        Log.e("MainActivity", message)
```

Comment:
Want this log still?

Reply:
Ya it helps with logging errors in the example. Otherwise it just says "An Error Happened" with no details.